### ISSUE TITLE
tinyMceConfig documentation clarity issues

### ISSUE THREAD

**Issue by ault** [umbraco/UmbracoDocs#2456]
***This is the page with issues: https://github.com/umbraco/UmbracoDocs/edit/master/Reference/Config/tinyMceConfig/index.md***

**Control Characters:** @ makes reference to **Extended Validation Elements**. Further down in that section it says they Extended Validation Elements are not working in the current version of Umbraco. 

Which is true? I do not know so will not edit this copy.

Also in the **Control Characters:** section: Neither I nor the people I work with could parse the copy for the "+" and "-" symbols.  I can guess at what they might mean, but other interpretations have been suggested.

**Comment by sofietoft** [comment]
Thanks for the report @ault ! 💪 

The article on the tinyMCE Config hasn't been verified and updated for Umbraco 8 as of yet.
So, what's in the article applies to Umbraco 7, and might not apply to Umbraco 8.
See: #2448 and #2095 

Could that perhaps have something to do with the issues you're having? 🤔

**Comment by sofietoft** [comment]
Hi @ault !

The article on the tinyMCE has recently been updated for Umbraco 8.
Would you be up for checking it out, and see if it answers the questions you've posted here? 😄 

You can find the article here: https://our.umbraco.com/Documentation/Reference/Config/tinyMceConfig/

**Comment by marcemarc** [comment]
@ault the confusion with the control character + Extended Validation Elements reference - was because that came from TinyMce docs - but in Umbraco, the Extended Validation Elements is used by Umbraco for core things like Macros being embedded in the RTE. Have updated that to hopefully make it clearer? eg you can use it, but if you do you'll break Umbraco, unless you keep it's config in there too.

The + and - control characters explanation also comes directly from the tinymce docs

here goes:
the **-** is straightforward, it just means if the contents of a tag are empty, it can be removed by tinyMCE
eg -h2, would mean
<h2></h2>
would be removed from the html

the + is less clear, but is about identifying tags that are 'ok' if they don't have child content.
eg if you were to have <div id="googlemap"></div> inserted into RTE, the <div is in theory empty and could be stripped but if the div is identified as being 'ok' to be blank you could have +div[id|style] that would mean it's ok to have a div in there with an Id or style attribute but with no content...

or at least that's my understanding! - not sure if it's more nuanced than that - TinyMce might explain it better!

**Comment by sofietoft** [comment]
Hi all 👋 
Apologies for the long wait on this one.

As the article about the tinyMceConfig in Umbraco has been updated - thanks @marcemarc , I'll go ahead and close this issue for now.

Please feel free to re-open, or to create a new issue, should any unanswered questions remain.
Thanks!
